Modix.Data/Models/Core/GuildUserBrief.cs

```diff
@@ -34,7 +34,13 @@ public class GuildUserBrief
         /// The name to display for this user.
         /// </summary>
         public string DisplayName
-            => Nickname ?? $"{Username}#{Discriminator}";
+            => Nickname ?? FullUsername;
```

Comment:
If `Username#Discriminator` is what we wanna display everywhere in the app, why not just re-purpose `DisplayName` and ensure that it's used everywhere?

Reply:
Agreed, I think we want `Username#Discriminator` to *be* the display name. Though since we define an extension method that does this, we can probably just remove both `DisplayName` and `FullUsername` from this class entirely. As a plus, that'd keep presentation logic out of the data layer.

Reply:
I'll investigate removing them entirely, seems reasonable - but there's definitely no reason to have two separate properties now.